ci/test/00_setup_env_native_cxx20.sh

```diff
@@ -13,5 +13,5 @@ export CPPFLAGS="-DDEBUG_LOCKORDER -DARENA_DEBUG"
 export PYZMQ=true
 export RUN_FUNCTIONAL_TESTS=false
 export GOAL="install"
-export BITCOIN_CONFIG="--enable-zmq --enable-reduce-exports --enable-crash-hooks --enable-c++20"
+export BITCOIN_CONFIG="--enable-zmq --enable-reduce-exports --enable-crash-hooks"
```

Comment:
this changes are super-seed this commit: https://github.com/dashpay/dash/pull/5971/commits/ebc569555952f18151ffac05397d413499a0c47f

Reply:
huh?

Reply:
979a8c901cfdbb649a92a00eb9c4fdc87a66b302: should probably rename `linux64_cxx20` job (and `ci/test/00_setup_env_native_cxx20.sh` file) now